meminfo.c

```diff
@@ -56,17 +57,25 @@ static long long available_guesstimate(const char* buf)
     return MemFree + Cached + Buffers - Shmem;
 }
 
-/* Parse /proc/meminfo.
+/* Parse /proc/meminfo and other related files:
+ *   * ZFS's /proc/spl/kstat/zfs/arcstats, if it exists
```

Comment:
Make this a separate function (`parse_arcstats`?) please

Reply:
Donel